pinot-broker/src/main/java/org/apache/pinot/broker/grpc/BrokerGrpcServer.java

```diff
@@ -181,15 +182,17 @@ public void submit(Broker.BrokerRequest request,
     try {
       sqlNodeAndOptions = RequestUtils.parseQuery(query, requestJsonNode);
     } catch (Exception e) {
-      // Do not log or emit metric here because it is pure user error
```

Comment:
In this block, only `SQL_PARSING` error can be thrown. So a simpler change is to increment the specific meter ?

Reply:
yes in this block currently only SQL_PARSING can be thrown. made the change primarily to keep the flow consistent with all other changes where we send back a brokerResponse (sort of making it future proof if more error codes are added in this block)